draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2233,7 +2169,8 @@ These suites share the following parameters:
 - m: 1
 - H: SHA-512
 - L: 72
-- f: Icart's method, {{icart}}
+- f: Simplified SWU method, {{simple-swu}}
+- Z: -2
```

Comment:
Isn't `Z = -1` the value that meets the criteria given in `{{simple-swu}}`? -2 also works, but I think is not consistent with that part of the document.

Is the idea to avoid -1 for reasons that Bjorn Haase has pointed out? If so, I think we should discuss that change in a separate PR, and this PR should stay consistent with the rest of the document.